docs/awesome-pets/papers/applications/set/psi.md

```diff
@@ -15,6 +15,10 @@ Note: one paper may be included in several categories (e.g. a paper may introduc
 
 ## PKC-based PSI
 
+- Laconic Private Set-Intersection From Pairings
+  *Aranha D, Lin C, Orlandi C, et al*
+  CCS 2022, [eprint](https://eprint.iacr.org/2022/529)
```

Comment:
We are currently ordering paper by "published year", can you move it down by one entry?

Reply:
Also you may add two spaces at the end of line 18-19, it'll look nicer on Github. Thanks